docs/index.md

```diff
@@ -0,0 +1,64 @@
+# LLVM remote index server
+This server provides about a day old index for
+[llvm-project](https://github.com/llvm/llvm-project/) to be used by
+clangd. It aims to lower resource usage of clangd on workstation machines by
+moving project-wide indexing and serving pieces to a centralized server.
+
+To make use of this service you need a clangd build with remote index support
+and point clangd at this particular index server.
+
+[Details about remote index](https://clangd.llvm.org/remote-index.html).
```

Comment:
nit: `More details on remote index internals`? This is also fine, but looks unexpected-ish as a single separate paragraph.